mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -125,7 +128,8 @@ public void initialize(@NonNull Context context, @NonNull String accessToken,
    * @param context     The context associated with the application
    * @param accessToken The accessToken associated with the application
    */
-  public void initialize(@NonNull Context context, @NonNull String accessToken, @NonNull String userAgent) {
+  public void initialize(@NonNull Context context, @NonNull String accessToken, @NonNull String userAgent,
```

Comment:
We're breaking the public API here and that would mean semver. We should implement this differently to not break backwards compatibility.

Adding a new overloaded `initialize` method should do the trick. 👀 https://github.com/mapbox/mapbox-java/blob/master/mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java#L109-L120